factcast-site/documentation/content/concept/_index.md

```diff
@@ -16,40 +16,34 @@ weight = 15
-When subscribing, the Consumer sends a specification of Facts he is interested in and might have received Facts in the past.
+{{%alert theme="danger"%}} TODO see `SubscriptionRequest` {{% /alert%}}
 
+In addition to the specification of events to read, the `SubscriptionRequest` also specifies the events to skip (e.g. due to previous consumption). The request also defines how to deal with `Facts` being published in the future.
 
-{{%alert theme="success" %}} *Note, that Facts are always guaranteed to be sent in the order published.* {{% /alert %}}
+{{%alert theme="success" %}} *Note that `Facts` are always guaranteed to be sent in the order they were published.* {{% /alert %}}
 
-The three usual subscription Models and their corresponding UseCases are:
+The three usual subscription models and their corresponding use cases are:
 
 | Subscription Type | Description |
 |:--|:--|
-| Follow | This is the 80% Use-Case for consumers. Here the Consumer does catch-up with Facts from the past and (after that) receives future Facts *as they are published*. <p>On subscribing, the consumer sends the 'id' of the last event processed and gets every Fact that matches his specification, that has been published *after* this last known Fact.</p>|
-| Catchup | <p>This subscription differs from Follow by completing, once the consumer has read the last of the currently published Facts.</p> <p>A usual Use-case for this kind of subscription is a write model, that needs to aggregate all information about a specific aggregate, in order to validate or reject an incoming command.</p>|
-| Ephemeral | The consumer *does not catch up* with Events that happened in the past, but only receives matching Facts *from now on*. <p>A good Use-Case is cache invalidation, but certainly not building Read Models.</p> |
+| Follow | This covers the 80% of the use cases. Here the consumer catches up with `Facts` from the past and also receives `Facts` in the future *as they are published*. <p>On subscription the consumer sends the `id` of the last event processed and gets every `Fact` that matches the specification and has been published *after* this last known `Fact`.</p>|
+| Catchup | <p>This subscription catches up with past events but does not receive any new `Facts` in the future.</p> <p>A usual use case for this subscription is a write model that needs to collect all kinds of information about a specific aggregate in order to validate or to reject an incoming command.</p>|
+| Ephemeral | The consumer does not catch up with past events, but receives matching `Facts` in the future. <p>A possible use case is e.g. cache invalidation. Not suitable for read models.</p> |
 
-Obviously all these subscription types rely on streaming transport which is implemented (at the time of writing) by GRPC.
+All these subscription typs rely on a streaming transport which uses (at the time of writing) GRPC.
```

Comment:
A little typo sneaked in. "typs" vs "types". Thx for contributing!

Reply:
Oh, sorry, thanks for fixing it!